Sources/XcodeGenKit/PBXProjGenerator.swift

```diff
@@ -56,10 +56,8 @@ public class PBXProjGenerator {
             try sourceGenerator.getFileGroups(path: group)
         }
 
-        let localPackages = Set(project.localPackages)
-        for package in localPackages {
-            let path = project.basePath + Path(package).normalize()
-            try sourceGenerator.createLocalPackage(path: path)
+        for (_, package) in project.localPackages {
+            try sourceGenerator.createLocalPackage(path: Path(package.path))
```

Comment:
We should sort the local packages by name here before generating them, so the output is deterministic